app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def boilerplate(array)
```

Comment:
could we rename this to something a little more specific?

Reply:
`to_options` next time we have one called `to_bar` :beer: